INSTALL.md

```diff
@@ -14,7 +18,10 @@ $ kubectl label nodes nodeName coredump=true
 run the following command:
 ```shell
 $ cd gencerts
-$ ./make-ca-cert.sh <ip of the host where the coredump-detector is deployed, and kube-apiserver will use this ip to access coredump-detector>
+$ ./make-ca-cert.sh webhook.default.svc <replace this with 1.[alone]- ip of the host where the coredump-detector is deployed, and kube-apiserver will use this ip to access coredump-detector 2.[as service]- the wenhook service DNS name>
+```
+If you choose setup [alone], you should modify `make-ca-cert.sh`, set `use_cn=false`.
```

Comment:
Why?

Reply:
set to `true` means allow to use dns name.